### DISCUSSION TITLE
No Privacy or Developer contact info for Edge Addon?

### DISCUSSION THREAD

**Discussion by soniariv** [microsoft/MicrosoftEdge-Extensions#138] [Q&A]
Good morning,
**What does it mean when there is no information about the Privacy or Developer contact info at an Add-on page?**
There is no Privacy contents disclaimer and no reference to any 'Contact' / email /website of the Software Developer for this Edge Add-on:
https://microsoftedge.microsoft.com/addons/detail/auto-refresher/lnehigejbkdlcjiocfndnbmigdfbegpb?hl=en
Anyone could publish an addon this way? How to trust such addons?
Thanks
soniariv

**Comment by soniariv** [comment]
No one?....

**Comment by ManikanthMSFT** [comment]
Hi @soniariv, we apologize for any inconvenience caused. We have received your query and are forwarding it to our team for further review. We will update you as soon as we have more information.

Thank you for your patience.

**Reply by ManikanthMSFT** [reply]
Hey @soniariv, please find the attached [documents ](https://learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/contact-extensions-team)which will address your concern.
